runbotics/runbotics-scheduler/src/scheduler-database/tenant/tenant.entity.ts

```diff
@@ -0,0 +1,41 @@
+import { Column, Entity, JoinColumn, ManyToOne, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+
+import { UserEntity } from '#/database/user/user.entity';
+
+import { Secret } from '../secret/secret.entity';
+import { ProcessContextSecret } from '../process-context-secret/process-context-secret.entity';
+import { ProcessContext } from '../process-context/process-context.entity';
+
+@Entity({ schema: 'public' })
+export class Tenant {
+    @PrimaryGeneratedColumn('uuid')
+    id: string;
+
+    @Column({ type: 'varchar', unique: true, nullable: false })
+    name: string;
+
+    @Column({ type: 'bigint', name: 'created_by' })
+    createdById: number;
+
+    @ManyToOne(() => UserEntity, user => user.tenants)
+    @JoinColumn({ name: 'created_by' })
+    createdBy: UserEntity;
+
+    @Column('timestamp without time zone')
+    created: Date;
+
+    @Column('timestamp without time zone')
+    updated: Date;
```

Comment:
this works?

Reply:
I've checked and this is how I use CreateDateColumn 
Thunderclient request:
![image](https://github.com/runbotics/runbotics/assets/69820069/eb3254aa-f553-4826-8ad7-5c505ceba00e)

Database:
![image](https://github.com/runbotics/runbotics/assets/69820069/94048379-01c3-4a4e-9535-a8ed755700ca)

Entitly:
`    @CreateDateColumn({ name: 'created_at', type: 'timestamp without time zone'})
    createdAt: Date;

    @ManyToOne(() => UserEntity)
    @JoinColumn({ name: 'created_by_id' })
    createdBy: UserEntity;

    @Column({ name: 'created_by_id' })
    createdById: number;

    @UpdateDateColumn({ name: 'updated_at', type: 'timestamp without time zone'})
    updatedAt: Date;

    @ManyToOne(() => UserEntity, user => user.updatedAttributes)
    @JoinColumn({ name: 'updated_by_id' })
    updatedBy: UserEntity;

    @Column({ name: 'updated_by_id' })
    updatedById: number;`

Service: 
![image](https://github.com/runbotics/runbotics/assets/69820069/bcb1a984-a28a-4c70-9952-97b79f099d7a)
 ----------- UpdateDateColumn
![image](https://github.com/runbotics/runbotics/assets/69820069/d1551f98-1a69-4b8d-949b-545d222da845)

![image](https://github.com/runbotics/runbotics/assets/69820069/c312f702-c3ad-4762-9e7b-088211827800)

![image](https://github.com/runbotics/runbotics/assets/69820069/12b417e0-6178-4571-85f7-597e8fb4f0e8)

So it works for both columns.

Reply:
Done.